src/app/pages/homepage/homepage.scss

```diff
@@ -86,6 +86,31 @@ $margin-promotion-sections-small: 15px;
   justify-content: center;
 }
 
+/** Carousel content styles */
+.docs-mat-card-divider {
+  width: 30%;
+  height: 5px;
+  margin: 15px auto;
+}
+
+.docs-carousel-item-container {
+  transition: transform 0.5s ease-in-out;
+  padding: 10px;
+  display: flex;
+}
+
+.docs-guides-card.mat-card {
+  text-align: center;
+}
+
+mat-card-title {
+  font-size: 20px;
+}
+
+a {
```

Comment:
be more specific, just using `a` may cause this style to get applied to elements you didn't intend for it to affect